pandas/tests/groupby/test_groupby.py

```diff
@@ -722,8 +734,11 @@ def test_omit_nuisance(df):
 
     # won't work with axis = 1
     grouped = df.groupby({'A': 0, 'C': 0, 'D': 1, 'E': 1}, axis=1)
-    result = pytest.raises(TypeError, grouped.agg,
-                           lambda x: x.sum(0, numeric_only=False))
+    msg = (r'\("unsupported operand type\(s\) for \+: '
+           "'Timestamp' and 'float'"
+           '"'r", 'occurred at index 0'\)")
```

Comment:
This is kind of tough to read - can you move the double quote to the end of the line before and have this one start with the `r` prefix?